### DISCUSSION TITLE
Preserve element order while parsing

### DISCUSSION THREAD

**Discussion by Neitex** [skrapeit/skrape.it#156] [Q&A] [answered]
**describe what you want to archive**
I want to parse some timetable web page, so right ordering of elements while parsing is crucial. It seems that skrape{it} does not keep order of elements. Is it somehow possible to parse HTML while preserving order of elements as they appear in HTML?

**Comment by christian-draeger** [comment, marked as answer]
It should already. Depending on your selector it will Traverse the Dom and return matching ordered as list.

could you provide some example code that show-case the issue? :) 

Nevertheless I will do some testing regarding the order if I am back home (in a few hours). I am also pretty sure that this is tested multiple times in unit and integration tests. I will check I am back to keyboard as well

**Comment by Neitex** [comment]
I'm sorry for taking your time. It seems that I somehow messed something up in my code, so it wasn't ordered right. Thank you and, again, sorry.

**Comment by christian-draeger** [comment]
No problem, here to help.
Just Let me know if you find issues or have questions :)
